conf/requirements.txt

```diff
@@ -8,3 +8,4 @@ python-constraint
 intervaltree
 jinja2
 pytest
+textx
```

Comment:
Why is this needed? My guess is because the fasm library isn't being installed with `pip` correctly so it isn't seeing the dependencies?

Reply:
Yes, that was the issue, removed textx